### PR TITLE
Create a6.is-cool.me.json

### PR DESCRIPTION
**Pull request by aaron2305**
## Requirements
- [x] You have completed your website.
- [x] The website is reachable.
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.

## Description

to live stream a football match or game 
## Link to Website

http://testingaktv.atwebpages.com/login.php

### CODE REVIEW

File: domains/a6.is-cool.me.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    "subdomain": "a6",
+
+    "owner": {
+        "username": "aaron2305",
+        "email": "aaronsmith2305@gmail.com"
+    },
+
+    "records": {
+        "CNAME": "testingaktv.atwebpages.com",
+
```
Comment (LightHostingFree) [inline domains/a6.is-cool.me.json:12]: add }, here

Comment (LightHostingFree) [reply domains/a6.is-cool.me.json:12]: And accept the CLA

**Review by LightHostingFree — CHANGES_REQUESTED** [review]

**Comment by aaron2305** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by LightHostingFree** [comment]
I have corrected your json file. Enjoy your free subdomain!

**Review by ShamyakGoel — APPROVED** [review]

**Review by LightHostingFree — APPROVED** [review]

**Pull request merged**